src/lib/components/searchbox/main.svelte

```diff
@@ -254,14 +254,14 @@
 					class:pb-1={last}
 					class:bg-neutral-100={current}
 					class:dark:bg-neutral-700={current}
-					class="rounded-lg"
+					class="px-4 py-0.5 rounded-lg"
 				>
 					<button
 						onmouseover={() => (currentIndex = i)}
 						onfocus={() => (currentIndex = i)}
 						onclick={() => (clickedIndex = i)}
 						type="submit"
-						class="px-4 py-0.5 size-full text-left"
+						class="size-full text-left"
```

Comment:
Was this change necessary? This makes it so you HAVE to click on the suggestion text for it to register the click, whereas before it was purposefuly made so you can click on the whole hovered area (left to the text, under the text, everything that has the lighter color in this img).
![image](https://github.com/hearchco/frontend/assets/31509435/77706672-3697-4103-8d3c-99163c9ba043)

Reply:
This was a misunderstanding and an honest mistake, figured it out over Discord. Pesky `class:pb-1={last}` I did not see at first was the problem which weirded me out. I changed it to `class:pb-0.5={last}` since it appears a lot less extreme. Hope that looks good to you as it does to me.